articles/active-directory-b2c/manage-user-accounts-graph-api.md

```diff
@@ -111,12 +111,14 @@ In user migration scenarios, if the accounts you want to migrate have weaker pas
 
 Every customer-facing application has unique requirements for the information to be collected. Your Azure AD B2C tenant comes with a built-in set of information stored in properties, such as Given Name, Surname, City, and Postal Code. With Azure AD B2C, you can extend the set of properties stored in each customer account. For more information on defining custom attributes, see [custom attributes (user flows)](user-flow-custom-attributes.md) and [custom attributes (custom policies)](custom-policy-custom-attributes.md).
 
-Microsoft Graph API supports creating and updating a user with extension attributes. Extension attributes in the Graph API are named by using the convention `extension_ApplicationObjectID_attributename`. For example:
+Microsoft Graph API supports creating and updating a user with extension attributes. Extension attributes in the Graph API are named by using the convention `extension_ApplicationObjectID_attributename`, where the `ApplicationObjectID` is the Object ID for the `b2c-extensions-app` application (found under **App Registrations** > **All Applications**). For example:
 
 ```JSON
 "extension_831374b3bd5041bfaa54263ec9e050fc_loyaltyNumber": "212342"
 ```
 
+Note that the Application Object ID has no dashes. Also, in order to access extension properties through the Microsoft Graph API, you must grant admin consent to the API permissions for the `b2c-extensions-app` application (found under **App Registrations** > **All Applications**).
```

Comment:
```suggestion
The the *Application (client) ID* as it's represented in the extension attribute name includes no hyphens. Also, in order to access extension properties through the Microsoft Graph API, you must grant admin consent in the **API permissions** of the `b2c-extensions-app` application (found under **App registrations** > **All Applications**).
```

Reply:
@nayrangnu @mmacy @msmimart
Admin consent is not required to the application which store extension attributes. It is required the application to read write user data.

```suggestion
Note that the the *Application (client) ID* as it's represented in the extension attribute name includes no hyphens. 
```

Here is my verification result

Create new application for storing extension attributes
![image](https://user-images.githubusercontent.com/28978715/87872885-711c3300-c9f7-11ea-9834-04516571a0bf.png)

There is no consent

![image](https://user-images.githubusercontent.com/28978715/87872889-78434100-c9f7-11ea-955b-aff69f204faf.png)

I can write extension attributes via custom policy. Also I can read it via MS Graph API.
![image](https://user-images.githubusercontent.com/28978715/87872892-7bd6c800-c9f7-11ea-9074-b7e7ab382e6e.png)

Reply:
CC: @yoelhor

Reply:
I think we add the note to the first sentence, as following/;  

Microsoft Graph API supports creating and updating a user with extension attributes. Extension attributes in the Graph API are named by using the convention `extension_ApplicationClientID_attributename`, where the `ApplicationClientID` is the **Application (client) ID** of the `b2c-extensions-app` application (found in **App registrations** > **All Applications** in the Azure portal). Note that the the *Application (client) ID* as it's represented in the extension attribute name includes no hyphens.  For example: